### PR TITLE
Add gwtools

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This adds a recipe for `gwtools`. This is extracted from #9687 to get it completed faster (this part of that build seems ok).

All co-maintainers have already posted on #9687 to indicate that they are happy being named in the recipe for this package.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@conda-forge/help-python, this recipe is ready for review. It is technically `noarch: python` but the windows builds will fail because the runtime dependencies are unix-only. Thanks.

File: recipes/gwtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+{% set version = "1.0.2" %}
+{% set sha256 = "ac413474d5ac3ddfbca21632d224a68a83b0a662ebdd215cb7e0391b92b79d40" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "{{ sha256 }}"
```
Comment (chrisburr) [inline recipes/gwtools/meta.yaml]: To make life easier for the update bot:
```suggestion
  sha256: ac413474d5ac3ddfbca21632d224a68a83b0a662ebdd215cb7e0391b92b79d40
```

File: recipes/gwtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{% set name = "gwtools" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "ac413474d5ac3ddfbca21632d224a68a83b0a662ebdd215cb7e0391b92b79d40" %}
```
Comment (chrisburr) [inline recipes/gwtools/meta.yaml]:
```suggestion
```

Comment (duncanmmacleod) [reply recipes/gwtools/meta.yaml]: done

**Pull request merged**